data/net/minecraft/core/Vec3i.mapping

```diff
@@ -85,9 +104,16 @@ CLASS net/minecraft/core/Vec3i
 		ARG 1 y
 	METHOD setZ (I)Lnet/minecraft/core/Vec3i;
 		ARG 1 z
+	METHOD south ()Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i one unit south.
 	METHOD south (I)Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i south by the distance.
 		ARG 1 distance
 	METHOD subtract (Lnet/minecraft/core/Vec3i;)Lnet/minecraft/core/Vec3i;
-		ARG 1 vector
+		COMMENT Creates a new Vec3i subtracted by the subtrahend.
+		ARG 1 subtrahend
```

Comment:
I'm really not a fan of this name, it *may* be technically correct (this is the first time I've ever heard the word and googling it says `quantity or number`, which I don't think a Vector specifically is (although I could be wrong.)

Either way, if someone has to google what it means, I don't think it is a good name.